deploy/chart/catalog_resources/components/chargeback.package.yaml

```diff
@@ -1,4 +1,4 @@
-#! package-manifest: ../../deploy/chart/catalog_resources/components/chargeback.v0.5.1.clusterserviceversion.yaml
+#! package-manifest: ./deploy/chart/catalog_resources/components/chargeback.v0.5.1.clusterserviceversion.yaml
```

Comment:
Changed the paths to be relative to project root, where `go run validator` is executed.